platform/ios/src/MGLMapboxEvents.m

```diff
@@ -99,7 +111,11 @@ - (instancetype)init {
 
 - (void)dealloc {
     @try {
-        NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Is there clarity on when to use standardUserDefaults vs `mme_configuration`? I wasn't positive which one to use here?